### PR TITLE
[SECURESIGN-200] Updating builder images to tags.

### PR DESCRIPTION
**Pull request by tommyd450**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder@sha256:98a0ff138c536eee98704d6909699ad5d0725a20573e2c510a60ef462b45cce0 AS build-env
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_9_1.21 AS build-env
```
[the remark this answers is not in the document]
Comment (Gregory-Pereira) [reply Dockerfile:16]: Could pull `sha`s like this locally with tag and `sha`.

**Comment by lance** [comment]
@tommyd450 this needs rebase

**Pull request closed without merging**